### ISSUE TITLE
Text color on map vs backgound

### ISSUE THREAD

**Issue by mthure** [fishlr/fishlr-web-support#5]
Hard to read the text on the depth curves when the sea depth is high.
Black text on dark blue background.

![map_text_color](https://cloud.githubusercontent.com/assets/19627562/15635031/023513b0-25d4-11e6-8aa6-874c4adcdff0.PNG)

**Comment by averas** [comment]
Good catch! The background is not actually the map, but the bathymetric data, so even though this is not a problem for all bathymetric styles (you can change style by clicking on the lake and go to bathymetry or by clicking on the bathymetry icon (the one that looks like a dart board) in the layers toolbar), this is still relevant for styles that fade into darkness or brightness with dark/bright curvature and labels.

This will be adressed with color ramped curvatures and labels which will be part of the user defined style epic.
